### DISCUSSION TITLE
What libmpv build do you use?

### DISCUSSION THREAD

**Discussion by TheBestPessimist** [mpvnet-player/mpv.net#609] [General]
As far as i can see, @stax76 uses [`zhongfly's`](https://github.com/zhongfly/mpv-winbuild/releases).
On the other hand, the [mpv website](https://mpv.io/installation/) recommends [shinchiro's](https://sourceforge.net/projects/mpv-player-windows/files) and that's what i used so far.
I assume there are other builds too.

What are the benefits/drawbacks of whichever build you use? _What's on the other side of the fence?_

---

_Note: Should this be made a poll?_

**Comment by stax76** [comment]
I think the builds of zhongfly and shinshiro are similar in regard of settings and build chain. The reason why I prefer now zhongfly's builds is that zhongfly is very active on both the mpv and mpv.net issue tracker. Similar to Patman making builds for StaxRip.

In the last beta, I missed using a newer build. My plan is using new mpv builds for mpv.net beta versions and stable mpv release builds for stable mpv.net versions.

I've always updated mpv and libmpv manually and really need to do this automated with a script.

**Comment by hooke007** [comment]
They are the same.
